.github/workflows/deploy.yaml.yml

```
@@ -0,0 +1,45 @@
+name: deploy.yaml
+on:
```

actionlint (1.7.10)

string should not be empty

(syntax-check)